pkg/solidity-utils/contracts/test/TestToken.sol

```diff
@@ -30,4 +30,12 @@ contract TestToken is ERC20, ERC20Burnable, ERC20Permit {
     function mint(address recipient, uint256 amount) external {
         _mint(recipient, amount);
     }
+
+    // burnWithoutAllowance was created to allow burn of token without approval. Example of use:
+    //
+    // MockGearboxVault.sol can't use burnFrom function (from ERC20Burnable) in unit tests, since
+    // MockGearboxVault doesn't have permission to burn relayer wrapped tokens and relayer is not a Signer
+    function burnWithoutAllowance(address sender, uint256 amount) external {
```

Comment:
I can't, because ERC20Burnable (inherited in TestToken) also implements a burn function with the same signature, and it conflicts with vault tests. I also tried to use burnFrom, from ERC20Burnable, but I had no success using that with Gearbox, because the relayer needed to authorize the transfer of wrapped tokens, and relayer is not a Signer in the tests.